src/NLog/Common/AsyncHelpers.cs

```diff
@@ -328,5 +328,29 @@ private static AsynchronousAction<T> ExceptionGuard<T>(AsynchronousAction<T> act
                 }
             };
         }
+
+        internal static bool WaitForDispose(this Timer timer, TimeSpan timeout)
```

Comment:
please document the return value.  Thanks!

Reply:
Added xml documentation